test/box-luatest/lua_memory_test.lua

```diff
@@ -0,0 +1,42 @@
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group('lua_memory setting test')
+
+local MB = 1024 * 1024
+local LUA_MIN_MEMORY = 256 * MB
+
+g.before_all(function(g)
+    g.server = server:new{
+        alias = 'default',
+        box_cfg = {lua_memory = LUA_MIN_MEMORY}
+    }
+    g.server:start()
+end)
+
+g.after_all(function(g)
+    g.server:drop()
+end)
+
+g.test_lua_memory_setting = function(g)
+    g.server:exec(function()
+        local MB = 1024 * 1024
```

Comment:
It is already declared as the upvalue above. This line can be dropped.

Reply:
It won't work because `g.server:exec` seems not to be able to capture the outer variable.